torch/csrc/autograd/python_variable.cpp

```diff
@@ -52,6 +51,91 @@ using namespace at;
 using namespace torch;
 using namespace torch::autograd;
 
+std::pair<py::object, py::dict> get_args_kwargs(const c10::OperatorHandle& op, const std::vector<c10::IValue>& arguments) {
+  py::gil_scoped_acquire g;
```

Comment:
it feels wrong to acquire GIL here, it should be precondition that GIL is held before you call this function